tool.go

```diff
@@ -480,7 +480,7 @@ func main() {
 		var root string
 
 		if len(args) > 1 {
-			cmdServe.Help()
+			cmdServe.HelpFunc()
```

Comment:
This will fix the build, but I don't think it's right.

Look at the definition of `HelpFunc` method:

https://godoc.org/github.com/spf13/cobra#Command.HelpFunc

It returns a function with default help behavior `func(*Command, []string)`, which should be called.

See some usage examples of it being called here:

https://sourcegraph.com/github.com/spf13/cobra/-/info/GoPackage/github.com/spf13/cobra/-/Command/HelpFunc